demo/kubernetes/1.5/mysql.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: v1
```

Comment:
any of these files that are duplicated between 1.5 and the main demo/kubernetes folder should be symlinked instead, which will reduce the duplication.